scripts/fish_pixels.py

```diff
@@ -0,0 +1,91 @@
+"""Goes through the fish images for manually marking the ends of the fish"""
+
+import cv2
+import os
+import shutil
+from util import data_path    
+
+file_path = "/Users/georgiamartinez/rov-vision-22/scripts/fish_pixels_list"
+
+file = open(file_path, "a")
+num_clicks = 0
+
+def record_mouse_position(event, x, y, flags, param):
+    """Records mouse position on click""" 
+
+    global num_clicks
+
+    label = param[0]
+    image_name = label+"/"+param[1]
+
+    # Get mouse coordinates
+    if event == cv2.EVENT_LBUTTONDOWN:
+        coords = f" {x} {y}"
+
+        # Format coordinates on the text file
+        if num_clicks % 2 == 0:
+            if num_clicks != 0:
+                file.write("\n")
+                print()
+
+            file.write(image_name)
+            print(image_name, end="")
+
+        file.write(coords)
+        print(coords, end="")                
+
+        num_clicks += 1
+
+def browse_images(label):
+    """Browse images with the 'a' and 'd' keys"""
+
+    global file, num_clicks
+
+    # Get list of images to browse through
+    image_path = os.path.join(data_path, "stereo", "1undistort", label)
```

Comment:
I think it would be better to just take `image_path` as the argument for this function. For the purposes of one-off scripts like this it's fine to have to go through the source code and change some constants, but it would be easier if the constants that need to change are outside of function definitions.